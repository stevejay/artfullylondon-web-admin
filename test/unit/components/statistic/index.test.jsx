import React from 'react'
import { shallow } from 'enzyme'

import Statistic from '_src/components/statistic'

it('should render correctly', () => {
  const wrapper = shallow(<Statistic entityType='venue' count={300} />)
  expect(wrapper).toMatchSnapshot()
})
