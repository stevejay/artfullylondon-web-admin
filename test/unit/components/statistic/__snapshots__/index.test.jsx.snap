// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  styleName="container-venue"
>
  <h2>
    Venue
  </h2>
  <p>
    300
  </p>
</div>
`;
